src/test/java/test/guice/jitbinding/FirstModuleTest.java

```diff
@@ -0,0 +1,27 @@
+package test.guice.jitbinding;
+
+import com.google.inject.AbstractModule;
+import org.testng.annotations.Guice;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Guice(modules = FirstModuleTest.FirstModule.class)
+public class FirstModuleTest {
```

Comment:
Can you confirm that your tests are included in the suite?
I think they should be added near https://github.com/cbeust/testng/blob/master/src/test/resources/testng.xml#L762-L769

Reply:
But the recommended way to write tests for TestNG itself is having samples (like yours) and having a test case that runs and asserts. See https://github.com/cbeust/testng/tree/master/src/test/java/test/name

Reply:
good catch, the tests weren't running without editing `testng.xml`.  changed to use one test that runs two samples, let me know if I misunderstood that suggestion.